Corrige l’utilisation et la documentation de \codedest

milouse opened the pull request:
Ces deux commits apportent les corrections suivantes :

- La documentation actuelle déclare dans la section 1.5 Information sur le destinataire « \codeclient Il est possible d’affecter un code au client, via la commande \codeclient{⟨code⟩}. » Je pense qu’il s’agit d’une erreur et que l’on parle ici de la commande `\codedest`. En effet la commande `\codeclient` est utilisée pour définir le texte utilisé en préfixe de ce même code client (par défaut "Code client :")
- La documentation (et le code) semble admettre que \codedest est sensé être optionnel. Malheureusement, sans \codedest la compilation échoue avec l’erreur ci-après. Je propose un correctif en déplaçant convenablement les marqueurs de fin de ligne `\\`.

```
! LaTeX Error: There's no line here to end.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.67 \entete
```

maieul commented:
Merci,

je n'ai plus trop l'utilité de cette classe. Seriez vous prêt à en reprendre la responsabilité (gestion de release, etc) ?

milouse commented:
Pourquoi pas. Je ne promets pas un suivi hyper assidu, mais pour l’instant je vais en avoir l’usage donc pas de souci pour m’ajouter en tant que mainteneur.

Pour information, je suis l’auteur d’un autre paquet LaTeX (latex-awesomebox). J’avais répondu à l’appel de l’association Gutemberg (sans en être membre) pour transférer mon dépôt dans leur "[organisation](https://github.com/gutenberg)". Ça permet de centraliser un peu les ressources, la visibilité etc. Une première action pourrait être de faire de même pour ce paquet ?

maieul commented:
Cela sera toujours mieux que le mien. Je n'ai pas le courage/l'usage de maintenir, et je préfère concernter mes forces tex sur les quelques packages où j'ai une vrai plus value. 

Oui, pourquoi pas, si cela vous arrange.

J'envoie un mail à Gutemberg et au CTAN pour informer du transfert ?

milouse committed:
Assure que \codedest est bien optionnel

Évite un crash avec le message d’erreur suivant:

! LaTeX Error: There's no line here to end.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.67 \entete

milouse committed:
Corrige la documentation à propos de \codedest

En effet, \codeclient est la commande utilisée pour générer le texte
préfixé (par défaut "Code client :")

milouse committed:
Évite un crash dans la règle par défaut du Makefile

Le fichier README a été renommé en README.md il y a quelques temps

See: 92a63cd792fe1417b4fec75afa6920b20fdd9ad1

milouse commented:
Je vous propose de continuer la discussion en privé par mail :)
